src/Plugin/views/row/EntityRowWithDynamicFormat.php

```diff
@@ -48,7 +48,6 @@ public function getViewModeOverride(ViewExecutable $view, DisplayPluginBase $dis
    * @return array
    */
   public function getFormatOptions() {
-    return \Drupal::entityManager()->getViewModeOptions($this->entityTypeId);
+    return \Drupal::entityTypeManager()->getViewModeOptions($this->entityTypeId);
```

Comment:
Does It work ? 
I didn't find `getViewModeOptions` in  https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21EntityTypeManagerInterface.php/interface/EntityTypeManagerInterface/9.0.x ?

Reply:
Addressed in the latest commit.